MediaBrowser.Model/Providers/RemoteLyricInfo.cs

```diff
@@ -0,0 +1,24 @@
+using MediaBrowser.Model.Lyrics;
+
+namespace MediaBrowser.Model.Providers;
+
+/// <summary>
+/// The remote lyric info.
+/// </summary>
+public class RemoteLyricInfo
```

Comment:
I think it would be worth adding the actual lyrics as well. This can be used to "preview" the lyrics when searching lyrics manually during playback.